.ci/pipeline_definitions

```diff
@@ -28,7 +16,7 @@ gardener-website-generator:
           vars:
               PUBLISH: "'true'"
               LOCAL_BUILD: "'false'"
-          publish_to: ['gardener_website', 'gardener_documentation']
```

Comment:
this is a change in behaviour as well.
`publish_to` attribute references (git) repositories the pipeline cloned via `repos: [...]`. in all referenced repositories a commit and push is performed via concourse, e.g. [here](https://concourse.ci.gardener.cloud/teams/gardener/pipelines/gardener-website-generator-master/jobs/master-head-update-job/builds/806#L67006f48:13).

by removing gardener/documentation repository changes made in pipeline are no longer pushed to said upstream repository